src/admin/migrations/m190624_112612_news_article_scheduler.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use yii\db\Migration;
+
+class m190624_112612_news_article_scheduler extends Migration
+{
+    public function safeUp()
+    {
+        $this->addColumn('news_article', 'is_online', $this->boolean()->notNull()->defaultValue(false));
+        $this->execute(<<<SQL
+UPDATE news_article
+SET is_online = (timestamp_display_from < NOW() AND (
+    NOT is_display_limit OR (
+        is_display_limit AND timestamp_display_until > NOW()
+    )
+));
+SQL
+        );
+        // TODO add scheduler entries?
```

Comment:
@nadar you might want to adjust the migration to convert existing timestamps to schedulers....

Reply:
if we release under version 2.0 i don't think it's a problem.